src/components/App.css

```diff
@@ -0,0 +1,320 @@
+body {
+  background-color: #f5f6fa;
+}
+
+table {
+  width: 60%;
+  margin: 10% 20%;
+  border: 1px solid black;
+}
+
+th {
+  border: 1px solid black;
+  padding-bottom: 20px;
+}
+
+.book-form {
+  width: 75rem;
+  margin-top: 2.5rem;
+  margin-left: 6.25rem;
+  border-top: solid 1px #e8e8e8;
+  margin-bottom: 16rem;
+}
+
+.book-form h4 {
+  width: 10.875rem;
+  height: 1.5rem;
+  margin: 1.813rem 32.75rem 0 0;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 1.25rem;
+  letter-spacing: -0.18px;
+  color: #888;
+}
+
+.book-form form input {
+  width: 41.5rem;
+  height: 2.813rem;
+  margin: 1.188rem 2.125rem 0 0;
+  padding: 0.813rem 0 0.813rem 1.063rem;
+  border-radius: 4px;
+  border: solid 1px #e8e8e8;
+  box-sizing: border-box;
+}
+
+.book-form form select {
+  width: 17.813rem;
+  height: 2.813rem;
+  border-radius: 4px;
+  border: solid 1px #e8e8e8;
+  background-color: white;
+}
+
+.book-chapter button {
+  width: 11.5rem;
+  height: 2.063rem;
+  margin: 0.75rem 1.75rem 0.563rem 3.688rem;
+  padding: 0.438rem 1.188rem 0.5rem 1.375rem;
+  border-radius: 3px;
+  font-family: 'Roboto Slab', serif;
+  font-size: 0.813rem;
+  letter-spacing: 0.5px;
+  text-align: center;
+  color: #fff;
+  background-color: #0290ff;
+  border: none;
+}
+
+.book-form form button {
+  width: 11.5rem;
+  height: 2.813rem;
+  margin: 1.188rem 0 0 2rem;
+  padding: 0.801rem 1.188rem 0.886rem 1.375rem;
+  border-radius: 3px;
+  box-sizing: border-box;
+  border: none;
+  background-color: #0290ff;
+  font-family: 'Roboto Slab', serif;
+  font-size: 0.813rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
```

Comment:
- [ ] It would be better if we implement the CSS DRY rule as pointed out by the previous code reviewer. This makes our code base simple and easy to understand. For example; `font-stretch: normal;
  font-style: normal;
  line-height: normal;` has appeared more than once hence repeated